recipes/parquet-python/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "parquet-python" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/parquet/parquet-{{ version }}.tar.gz
+  sha256: fb1c90768c1b9159d4d6a9b3112ea8107b0b46d7491c5ac452ba7350f333bf0a
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

```suggestion
build:
  noarch: python
  number: 0
  script: {{ PYTHON }} -m pip install . --no-deps -vv
  entry_points:
    - parquet = parquet.__main__:main
```
The `.exe` was complained about created because pip creates them for console_scripts entry points on windows; if you provide conda-build with a specific list of entry_points, it knows that it needs to handle that issue for noarch packages on windows, so you can remain noarch.

---

>it knows that it needs to handle that issue for noarch packages on windows, so you can remain noarch.

Yes. Note that this is mentioned in the docs for noarch python

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

> I also believe --no-deps isn't required on conda-forge.

Agreed. I think there is some environment variable that sets this for all builds.